src/transformers/models/layoutlmv2/configuration_layoutlmv2.py

```diff
@@ -0,0 +1,171 @@
+            Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (:obj:`int`, `optional`, defaults to 3072):
+            Dimension of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_act (:obj:`str` or :obj:`function`, `optional`, defaults to :obj:`"gelu"`):
+            The non-linear activation function (function or string) in the encoder and pooler. If string,
+            :obj:`"gelu"`, :obj:`"relu"`, :obj:`"selu"` and :obj:`"gelu_new"` are supported.
+        hidden_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (:obj:`float`, `optional`, defaults to 0.1):
+            The dropout ratio for the attention probabilities.
+        max_position_embeddings (:obj:`int`, `optional`, defaults to 512):
+            The maximum sequence length that this model might ever be used with. Typically set this to something large
+            just in case (e.g., 512 or 1024 or 2048).
+        type_vocab_size (:obj:`int`, `optional`, defaults to 2):
+            The vocabulary size of the :obj:`token_type_ids` passed when calling :class:`~transformers.LayoutLMv2Model`
+            or :class:`~transformers.TFLayoutLMv2Model`.
+        initializer_range (:obj:`float`, `optional`, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        layer_norm_eps (:obj:`float`, `optional`, defaults to 1e-12):
+            The epsilon used by the layer normalization layers.
+        max_2d_position_embeddings (:obj:`int`, `optional`, defaults to 1024):
+            The maximum value that the 2D position embedding might ever be used with. Typically set this to something large
+            just in case (e.g., 1024).
+        max_rel_pos (:obj:`int`, `optional`, defaults to 128):
+            The maximum number of relative positions to be used in the self-attention mechanism.
+        rel_pos_bins (:obj:`int`, `optional`, defaults to 32):
+            The number of relative position bins to be used in the self-attention mechanism.
+        fast_qkv (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not to use a single matrix for the queries, keys, values in the self-attention layers.
+        max_rel_2d_pos (:obj:`int`, `optional`, defaults to 256):
+            The maximum number of relative 2D positions in the self-attention mechanism.
+        rel_2d_pos_bins (:obj:`int`, `optional`, defaults to 64):
+            The number of 2D relative position bins in the self-attention mechanism.
+        convert_sync_batchnorm (:obj:`bool`, `optional`, defaults to :obj:`True`):
+            Whether or not to convert BatchNorm layers to SyncNorm layers before wrapping the visual backbone with DDP.
```

Comment:
This parameter is probably to be removed?

Reply:
Maybe replaced by a method on the models? Or does it need to be accessed before intializing it?

Reply:
It is required when initializing the visual backbone.